playground/frontend/pubspec.yaml

```diff
@@ -35,7 +35,7 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 version: 1.0.0+1
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ">=2.17.0 <3.0.0"
```

Comment:
We have already upgraded to Flutter 3. It requires Dart 2.17+ so it was used anyway.